src/App.tsx

```diff
@@ -1,13 +1,16 @@
 import { BrowserRouter } from 'react-router-dom';
 
+import { PouchDBProvider } from 'context';
 import { Router } from './routes';
 import './App.scss';
 
 const App = (): JSX.Element => {
   return (
-    <BrowserRouter>
-      <Router />
-    </BrowserRouter>
+    <PouchDBProvider>
+      <BrowserRouter>
```

Comment:
Provide a base name to the react router so that all the router navigations are strictly internal, without having to specify the PUBLIC_URL for very route/navigation.

Reference: https://create-react-app.dev/docs/deployment/#building-for-relative-paths
```suggestion
      <BrowserRouter basename={ process.env.PUBLIC_URL }>
```

Reply:
Oh got it.